lib/src/external_module.cc

```diff
@@ -313,8 +315,10 @@ ActionResponse ExternalModule::callBlockingAction(const ActionRequest& request)
     LOG_TRACE("Input for the %1%: %2%", request.prettyLabel(), action_args);
 
     auto exec = lth_exec::execute(
-#ifdef _WIN32
+#if defined(_WIN32)
         "cmd.exe", { "/c", path_, action_name },
+#elif defined(__sun)
+        "/usr/bin/ctrun", { "-l", "child", path_, action_name },
```

Comment:
My opinion is that for stuff where we know it'll always be in the same place we should just use absolute paths. I use `which` when either the command might not exist, or if it might be in different places depending on the particular OS flavor.

For ctrun in solaris, I think we can just use an absolute path